### PR TITLE
add ikki@0.2.0 w/ npm auto-update

### LINKED ISSUE #9263: [Request] Ikki
**Library name:** Ikki 
**Git repository url:** https://github.com/cognitom/ikki
**npm package url(optional):** https://www.npmjs.com/package/ikki
**License(s):** MIT
**Official homepage:** http://cognitom.github.io/ikki/
**Wanna say something? Leave message here:**

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #9263
Related issue(s): #9414 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/cognitom/ikki
 * Official website (optional, not the repository): http://cognitom.github.io/ikki/
 * NPM package url (optional): https://www.npmjs.com/package/ikki
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 4
   - Count of watchers: 48
   - Count of forks: 3
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 12
   - Downloads in the last month: 28

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9263, cc @cognitom #9414

### CODE REVIEW

**Comment by qwe19272375** [comment]
Library repository detail added. @PeterDaveHello please help me review it, thanks!

**Comment by PeterDaveHello** [comment]
You should ask review from your mentor @qwe19272375

**Comment by qwe19272375** [comment]
@nolsherry Please help me review this PR, thanks!

**Comment by nolsherry** [comment]
@qwe19272375 It seems that `riot-dev.js` is not minified,  please minify all files under `dist` folder, and you can use command `~/web-minify-helper/minify.sh` in `ajax/libs/ikki/` folder to minify all files under `dist` folder, thanks!

**Comment by PeterDaveHello** [comment]
I don't think we need to minify `riot-dev.js`.

**Comment by nolsherry** [comment]
@PeterDaveHello thanks for your  advice.
@qwe19272375 please just modify your commit message with following format, thanks!
`close #9263, cc @cognitom, cc #9414`

**Comment by qwe19272375** [comment]
Commit message modified.

**Comment by nolsherry** [comment]
LGTM, @x09326 please help me review this PR, thanks!

**Comment by qwe19272375** [comment]
> PeterDaveHello commented 7 days ago
I don't think we need to minify riot-dev.js.

Should I minify the `riot-dev.js` ?

**Comment by PeterDaveHello** [comment]
Nope, actually, I don't think we need to add `riot-dev.js`, would you please remove it?

**Comment by qwe19272375** [comment]
Remove `riot-dev.js` and modify `package.json`. @nolsherry Please help me review it, thanks!

File: ajax/libs/ikki/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "router",
+    "riotcomponent",
+    "component"
+  ],
+  "npmName": "ikki",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "ikki*"
```
Comment (PeterDaveHello) [inline ajax/libs/ikki/package.json]: Let's use `**/*ikki*`

**Pull request merged**